packages/commonwealth/client/scripts/hooks/useAnimation.ts

```diff
@@ -1,18 +1,24 @@
 import useNecessaryEffect from 'hooks/useNecessaryEffect';
 import { useState } from 'react';
 
-export const useAnimation = () => {
+interface useAnimationProps {
+  transitionDuration?: string;
+}
+
+export const useAnimation = ({
+  transitionDuration = '1s',
+}: useAnimationProps = {}) => {
   const [animationStyles, setAnimationStyles] = useState({
     opacity: 0,
-    transform: 'translateY(40px)',
-    transition: '1s',
+    transform: 'translateY(100%)',
```

Comment:
this is used in other places too, so you can't change the value here as it will broke other places. Probably you should make this value dynamic, similarly as you did for transitionDuration

Reply:
@masvelio Good catch, changing that now. Thanks

Reply:
@masvelio Just pushed those changes